diff -up wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/rfbproto.c.orig wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/rfbproto.c
--- wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/rfbproto.c.orig	2024-07-27 12:00:00.000000000 +0000
+++ wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/rfbproto.c	2024-07-27 12:00:01.000000000 +0000
@@ -54,7 +54,7 @@
 #include "crypto.h"
 #include "cursor.h"
 
-#include "minilzo.h"
+#include <lzo/lzo1x.h>
 
 /*
  * NOTE: we assume a little-endian, 32 bit pixel format to be used
diff -up wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/encodings/ultra.c.orig wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/encodings/ultra.c
--- wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/encodings/ultra.c.orig	2024-07-27 12:00:00.000000000 +0000
+++ wlvncc-860232f34a77837915a94078efe8cd527fa582e3/src/encodings/ultra.c	2024-07-27 12:00:01.000000000 +0000
@@ -25,7 +25,7 @@
  * USA.
  */
 
-#include "minilzo.h"
+#include <lzo/lzo1x.h>
 
 #define ULTRA_BUFFER_SIZE (64 * 1024)
 #define ULTRA_MAX_SIZE(w, h) ((((w + 15) / 16) * ((h + 15) / 16)) * 3)
